Aries/Aries/Admin/Admin.swift

```diff
@@ -15,17 +15,21 @@ public class Admin {
     private var storage: Storage
     private var events: AriesEvents
     private var agentConnections: AriesConnections
+    private var agentProofs: AdminProofs
+    private var agentCredentials: AdminCredentials
     private var adminInvitationUrl: String?
     public var connections: AdminConnections? = nil
     
     //connectToAdmin data for reference in event handling... there might be a better way to do this?
     private var adminConnectionId: String? = nil
     private var adminCompletion: (_ result: Result<Void, Error>)->Void
     
-    public init(messageSender: MessageSender, storage: Storage, connections: AriesConnections, events: AriesEvents, adminInvitationUrl: String? = nil){
+    public init(messageSender: MessageSender, storage: Storage, connections: AriesConnections, proofs: AdminProofs, credentials: AdminCredentials, events: AriesEvents, adminInvitationUrl: String? = nil){
```

Comment:
```suggestion
    public init(messageSender: MessageSender, storage: Storage, connections: AriesConnections, events: AriesEvents, adminInvitationUrl: String? = nil){
```
Admin submodules should be constructed in the setAdminConnection function in the Admin module like done here: https://github.com/Indicio-tech/aries-mobile-agent-ios/blob/b57a0c104406bdc90bda6e29881bf61252f94a7e/Aries/Aries/Admin/Admin.swift#L99